arangod/GeneralServer/CommTask.cpp

```diff
@@ -185,11 +185,11 @@ CommTask::Flow CommTask::prepareExecution(auth::TokenCache::Entry const& authTok
     }
     [[fallthrough]];
     case ServerState::Mode::TRYAGAIN: {
+      // the following paths are whitelisted on followers
```

Comment:
I have another PR open (to be merged soon) that tries to get rid of white- and blacklisting terminology in our code.
```suggestion
      // the following paths are allowed on followers
```